### ISSUE TITLE
Input of different size

### ISSUE THREAD

**Issue by SuJingZhi** [fchollet/keras#6262]
Hello everyone：
I am a newer to keras. I want to use keras classify the image. There are some problem. I have two kinds of inputs which are paired but have different size，so I want to use the functional model. But I wonder if the two different inputs will be not paired after shuffle. And any one have an idea to deal with the problem?
Thank you!

**Comment by nijianmo** [comment]
Hi. What do you mean by different size? Different batch size or different shape of each input sample? 

For different shape of each input sample, I think keras can handle that.
